bake/bake.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/docker/buildx/util/buildflags"
 	"github.com/docker/buildx/util/platformutil"
 	"github.com/docker/cli/cli/config"
-	"github.com/docker/docker/pkg/urlutil"
+	"github.com/docker/docker/builder/remotecontext/urlutil"
```

Comment:
We should look at replacing this with something from BuildKit, e.g. https://github.com/moby/buildkit/blob/df49b648c8bfef657ba99543390516a3f518d22a/util/gitutil/git_protocol.go#L18


And/or move this package to BuildKit.


The documentation I put in the packege may be nice to preserve though, so if there's no equivalent in BuildKit; perhaps at least copy the docs; https://github.com/moby/moby/blob/7624f8aeb1836cd75a0912d9614930e8a59eaf4e/builder/remotecontext/urlutil/urlutil.go#L24-L75